lxc/config.go

```diff
@@ -55,6 +55,9 @@ func (c *configCmd) usage() string {
 			"lxc config edit <container>                            Edit container configuration in external editor\n" +
 			"lxc config get <container> key                         Get configuration key\n" +
 			"lxc config set <container> key [value]                 Set container configuration key\n" +
```

Comment:
I assume so that not mentioning a value would unset the key?  So yes if we have unset we can drop that.